### PR TITLE
🎉 ADD: define remote

### PR DESCRIPTION
**Pull request by shhac**
Quick description of `remote` as a concept

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Before we dive in, there are a few terms which should be mentioned first, since
   * If you checkout a branch, HEAD symbolically refers to that branch, indicating that the branch name should be updated after the next commit operation.
   *  If you checkout a specific commit, HEAD refers to that commit only. This is referred to as a detached _HEAD_, and occurs, for example, if you check out a tag name.
 
+* **remote** — This is a reference to a different copy of the same repository located elsewhere, e.g. on another computer. The **remote** repository may be in a different state to the local copy but should always share at least one common ancestor commit. That the local and remote repositories can diverge from each other is what allows `git` to enable collaborative distributed work.
```
Comment (matte) [inline index.md:25]: Great addition.  One sentence tweak:

>That the local and remote repositories can diverge from

Local and remote repositories use this common commit to diverge ...

Comment (shhac) [reply index.md:25]: Not certain how you mean for that to join with the latter half of the sentence, something like this?

```suggestion
* **remote** — This is a reference to a different copy of the same repository located elsewhere, e.g. on another computer. The **remote** repository may be in a different state to the local copy but should always share at least one common ancestor commit. Local and remote repositories use a common commit to diverge from each other, allowing `git` to enable collaborative distributed work.
```

Comment (matte) [reply index.md:25]: Goal was to start a sentence with article `That` was referring to.  I like the above change 🥇

**Review by jwiegley — APPROVED** [review]

**Comment by jwiegley** [comment]
Thank you!

**Pull request merged**